server/src/db/repositories/reportRepository.ts

```diff
@@ -16,23 +16,24 @@ const repository = () => {
   };
 
   const aggregateCompatibilityByProduct = (filters, sorting, callback) => {
```

Comment:
Please add types for this input and other functions in the reportRepository

Reply:
done